compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/TaintAnalyzer.java

```diff
@@ -2445,6 +2473,28 @@ private void analyzeInvocation(BLangInvocation invocationExpr) {
         getCurrentAnalysisState().taintedStatus = returnTaintedStatus;
     }
 
+    private boolean restoreTableIfIgnored(BInvokableSymbol invokableSymbol, Map<Integer, TaintRecord> origTaintTable,
+     TaintedStatus argumentTaintReturnValue) {
```

Comment:
Is this indentation correct here? I think we can change this latter

Reply:
Ack, will fix this.